website/content/docs/connect/cluster-peering/index.mdx

```diff
@@ -54,7 +54,7 @@ Not all features and functionality are available in the beta release. In particu
 
 - Mesh gateways for _server to server traffic_ are not available.
 - Services with node, instance, and check definitions totaling more than 4MB cannot be exported to a peer.
-- Some dynamic routing features, such as splits and custom routes, cannot target services in a peered cluster.
+- The `service-splitter` and `service-router` config entry kinds cannot directly target a peer. To split or route traffic to a service instance on a peer, you must supplement your desired dynamic routing definition with a `service-resolver` config entry on the dialing cluster.
```

Comment:
What do you think about linking to the "Service splitters and custom routes" section you wrote here? It feels like readers might think this is missing context.